eal/cluster_configuration.ex

```diff
@@ -33,6 +33,15 @@ defmodule AntikytheraEal.ClusterConfiguration do
     This callback is called only at startup and the return value is passed to `RaftFleet.activate/1`.
     """
     @callback zone_of_this_host() :: String.t()
+
+    @doc """
+    Returns the number of seconds for which the initial health check to this host is delayed to
+    provide ample warm-up time for the host.
+
+    This callback is called only at startup and the return value is used to calculate the retry count for
+    establishing connections to other nodes.
+    """
+    @callback health_check_grace_period() :: pos_integer
```

Comment:
```suggestion
    @callback health_check_grace_period() :: non_neg_integer
```

Reply:
Yes, we can set 0 as a health check grace period in AWS ASG.
Fixed in https://github.com/access-company/antikythera/pull/168/commits/8b5bafb82dd33b78e6180010ab3c6398cec7af95